static/templates/stream_settings/stream_settings.hbs

```diff
@@ -41,14 +41,12 @@
                 <h3 class="stream_setting_subsection_title">
                     {{t "Stream permissions" }}
                 </h3>
-                {{#if can_change_stream_permissions}}
                 <div class="stream_permission_change_info alert-notification"></div>
```

Comment:
This also makes us always add this alert-notification div, but it's an empty div so there's no difference in the normal flow... and in other cases, we probably intend for it to be there?

Reply:
Yeah, I think havin this (invisible) element be present in the HTML is what's intended.